### ISSUE TITLE
[12] Admin user (id=2) should not be archived.

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#27413]
Impacted versions: 12.0

Steps to reproduce: Users --> Archive Admin user

Current behavior: Admin can archive Admin user (Admin itself)

Expected behavior: Admin should not be archived
You cannot deactivate the admin user.
Video/Screenshot link (optional):

Ps:I think its is because of SUPERUSER_ID, As Current logged in user (Admin) is with id=2.

**Comment by pedrobaeza** [comment]
AFAIK, #31216 talked about deleting the user, not archiving it.

**Comment by sswapnesh** [comment]
True but the Idea was to prevent Archive/Unlink of Admin user.

**Comment by pedrobaeza** [comment]
Archiving it is not a problem.

**Comment by Yenthe666** [comment]
Indeed, archiving is no issue (and you might want to be able to do it actually as the user costs money on an enterprise version and you could technically go without it). The real issue is deleting.

**Comment by fmdl** [comment]
@Yenthe666 you are right you could archive ADMIN (uid = 2).

**Comment by C3POdoo** [comment]
Dear @sswapnesh,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
